contracts/Cartographer.sol

```diff
@@ -855,7 +847,11 @@ contract Cartographer is Ownable, Initializable, ReentrancyGuard {
                 _crossCompound,
                 false
             );
-        lastDepositTimestamps[msg.sender] = block.timestamp;
+
+        uint256 taxResetBP = isNativeFarmToken[_token] ? nativeTaxResetOnDepositBP : baseTaxResetOnDepositBP;
+        if (_amount > staked.mul(taxResetBP).div(10000)) {
```

Comment:
we don't need to use "mul" and "div" because we use solidity v8.0

Reply:
Ah right of course